### PR TITLE
fix: connection banner visible when on lock screen

### PR DESCRIPTION
**Pull request by estebanmino**
Fixes BX-####
Figma link (if any):

## What changed (plus any additional context for devs)

found this issue introduced here https://github.com/rainbow-me/browser-extension/pull/1404/files

https://github.com/rainbow-me/browser-extension/assets/12115171/2f02ba34-8d13-4a47-8b63-1a5d4437d260

you can see the connection banner for a second while the login screen is showing up

now is fixed, it doesn't appear

https://github.com/rainbow-me/browser-extension/assets/12115171/e409dd5d-9166-4ed2-96d6-5df0f58a9f22

### CODE REVIEW

**Review by BrodyHughes — COMMENTED** [review]
still think i'm seeing this:
<video src="https://github.com/rainbow-me/browser-extension/assets/41711440/062646a0-4ba8-4d2e-a46c-e5e618a79231">

File: src/entries/popup/components/AppConnection/AppConnectionWatcher.tsx [hunk 1]
```diff
@@ -110,6 +110,7 @@ export const AppConnectionWatcher = () => {
   }, []);
 
   const checkAndDisplayBanner = useCallback(() => {
+    console.log('- checkAndDisplayBanner');
```
Comment (BrodyHughes) [inline src/entries/popup/components/AppConnection/AppConnectionWatcher.tsx:113]: logzzz

**Review by BrodyHughes — APPROVED** [review]
lgtm!

File: src/entries/popup/components/AppConnection/AppConnectionWatcher.tsx [hunk 2]
```diff
@@ -163,14 +165,21 @@ export const AppConnectionWatcher = () => {
...
+      '-- location !== home',
+      location.pathname,
+      location.pathname !== ROUTES.HOME,
+    );
     if (location.pathname !== ROUTES.HOME) {
       shouldAnimateOut.current = true;
       hide();
     }
-  }, [hide, location.pathname]);
+  }, [hide, location, location.pathname]);
```
Comment (derHowie) [inline src/entries/popup/components/AppConnection/AppConnectionWatcher.tsx:182]: we can remove this, right?

**Comment by estebanmino** [comment]
@BrodyHughes @derHowie just removed the logz 🤦

**Pull request merged**